apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js

```diff
@@ -1,21 +1,162 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import styled from 'styled-components'
 
-const RequestAssignment = ({ issues, rate, onSubmit }) => (
-  <div>RequestAssignment</div>
-)
-
-RequestAssignment.propTypes = {
-  /** array of issues to allocate bounties on */
-  issues: PropTypes.arrayOf(
-    PropTypes.shape({
-      id,
-      level,
-    })
-  ),
-  /** base rate in pennies */
-  rate: PropTypes.number,
-  onSubmit: PropTypes.func,
+import {
+  Field,
+  Text,
+  TextInput,
+  theme,
+  Info,
+  SafeLink,
+  IconAttention
```

Comment:
Might not be needed

Reply:
IconAttention wasn't needed, the others are useful.